### PR TITLE
Move Migration to Docker to ASP.NET docs

### PR DESCRIPTION
**Pull request by spboyer**
Related to https://github.com/aspnet/Docs/issues/2679
Dependent on https://github.com/aspnet/Docs/pull/2877

@mairaw - do we also remove from ToC?

### CODE REVIEW

File: docs/framework/docker/aspnetmvc.md [hunk 1]
```diff
@@ -1,221 +1,8 @@
...
-keywords: Windows Containers, Docker, ASP.NET MVC
-author: BillWagner
-ms.author: wiwagn
-ms.date: 02/01/2017
-ms.topic: article
-ms.prod: .net-framework
-ms.technology: dotnet-mvc
-ms.devlang: dotnet
-ms.assetid: c9f1d52c-b4bd-4b5d-b7f9-8f9ceaf778c4
+redirect_url: /aspnet/mvc/overview/deployment/docker-aspnetmvc
```
Comment (mairaw) [inline docs/framework/docker/aspnetmvc.md]: we would want to preserve the comments when we migrate too, so add the following metadata as well:

`redirect_document_id: TRUE`

**Review by mairaw — CHANGES_REQUESTED** [review]
Yes, let's remove it from the TOC as well. Thanks!

**Comment by spboyer** [comment]
@mairaw - should be good to go now, once the ASP.NET items are merged.
cc:/ @tdykstra @Rick-Anderson

**Review by mairaw — APPROVED** [review]
I'll merge as soon as build is green. Thanks @spboyer!

**Pull request merged**